x-pack/plugins/apm/public/services/rest/tutorial_apm_fleet_check.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { callApmApi } from './createCallApmApi';
+
+export async function hasFleetApmIntegrations() {
```

Comment:
It would be better if you can co-locate all the tutorial code in a single folder, perhaps ` x-pack/plugins/apm/public/tutorial` containing `tutorial_apm_fleet_check.ts` and `/tutorial_fleet_instructions` (and an other tutorial related code.